docs/user/setup.rst

```diff
@@ -3,49 +3,65 @@
 Environment setup
 =================
 
-This is what you will be doing, in a nutshell:
+Scala Native has the following system requirements:
 
-* installation of sbt
-* installation of LLVM and Clang
-* installation of Boehm GC
+* Java 8 or later
+* sbt 0.13.15
```

Comment:
The `project/build.properties` file ensures that the build is reproducible across different system so it should always be created. http://www.scala-sbt.org/0.13/docs/Basic-Def.html#Specifying+the+sbt+version